templates/Web/Projects/ReactDefault/frontend/src/components/Footer/Footer.js

```diff
@@ -1,5 +1,5 @@
 ﻿import React from "react";
-import styles from "./footer.module.css";
+import styles from "./Footer.module.css";
```

Comment:
is this css a Component File?

Reply:
it is like this by convention https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

Reply:
The react css modules are written in PascalCase